### PR TITLE
Compress images and enable gzip

### PR DESCRIPTION
**Pull request by FanaHOVA**

### CODE REVIEW

File: views/layout.slim [hunk 1]
```diff
@@ -44,11 +41,15 @@ html
       /! /.container
     script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"
     link href="https://fonts.googleapis.com/css?family=Lato:400,400i,700,700i" rel="stylesheet" /
-    script src="assets/javascripts/bootstrap.min.js"
+    script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"
     script src="assets/javascripts/scripts.js"
```
Comment (skatkov) [inline views/layout.slim]: Recommend to try moving these script to bottom of a page. It will speed up initial page load dramatically.

**Review by skatkov — CHANGES_REQUESTED** [review]
Since we're talking about speed imrovements, i woudl suggest:
- Moving JS code to bottom of a page, so it would start loading after page is rendered.
- Changing dimension for face shoots - lady is 2800 pixels wide. Seems crazy good quality for a webpage :P

File: views/layout.slim (hunk 1 above)
Comment (FanaHOVA) [reply views/layout.slim]: @skatkov I don't understand, they already are at the bottom at the page?

Comment (FanaHOVA) [reply views/layout.slim]: @skatkov Ah you meant at the bottom of the list, got it!

**Review by skatkov — APPROVED** [review]

**Pull request merged**